libafl_frida/src/lib.rs

```diff
@@ -124,14 +135,25 @@ impl FridaOptions {
                     assert_eq!(
                         core_ids.len(),
                         1,
-                        "Client should only be enabled on one core"
+                        "[asan_cores] Client should only be enabled on one core"
                     );
                     let core_id = core_ids[0].id;
                     options.enable_asan = asan_cores.contains(&core_id);
                 }
             }
+            if options.enable_cmplog {
+                if let Some(cmplog_cores) = cmplog_cores {
+                    let core_ids = get_core_ids().unwrap();
+                    assert_eq!(
+                        core_ids.len(),
+                        1,
+                        "[cmplog_cores] Client should only be enabled on one core"
```

Comment:
Use the same message as above. No need to prefix with `[cmplog_cores]` or `[asan_cores]` imo.